examples/frequently_asked_question_bot/telegram/.env.example

```diff
@@ -0,0 +1 @@
+TG_BOT_TOKEN=
```

Comment:
I suggest we rename all the .env.example files to .env, because creating their own .env files can be too complicated for users